config/config.go

```diff
@@ -274,26 +274,13 @@ func NewConfig() (c *Config) {
 	// Prometheus configuration
 	c.ExternalServices.Prometheus.URL = strings.TrimSpace(getDefaultString(EnvPrometheusServiceURL, fmt.Sprintf("http://prometheus.%s:9090", c.IstioNamespace)))
 	c.ExternalServices.Prometheus.CustomMetricsURL = strings.TrimSpace(getDefaultString(EnvPrometheusCustomMetricsURL, c.ExternalServices.Prometheus.URL))
-	c.ExternalServices.Prometheus.InsecureSkipVerify = getDefaultBool(EnvPrometheusInsecureSkipVerify, false)
-	c.ExternalServices.Prometheus.Auth = strings.TrimSpace(getDefaultString(EnvPrometheusAuth, PrometheusAuthStrategyNone))
-	c.ExternalServices.Prometheus.CAFile = strings.TrimSpace(getDefaultString(EnvPrometheusCAFile, ""))
-	if c.ExternalServices.Prometheus.Auth != PrometheusAuthStrategyNone && c.ExternalServices.Prometheus.Auth != PrometheusAuthStrategyBearer {
-		log.Errorf("Unknown Prometheus Auth strategy. Valid options are %v or %v", PrometheusAuthStrategyNone, PrometheusAuthStrategyBearer)
-	}
+	c.ExternalServices.Prometheus.Auth = getAuthFromEnv("PROMETHEUS")
 
 	// Grafana Configuration
 	c.ExternalServices.Grafana.DisplayLink = getDefaultBool(EnvGrafanaDisplayLink, true)
-	c.ExternalServices.Grafana.InCluster = getDefaultBool(EnvGrafanaInCluster, true)
+	c.ExternalServices.Grafana.InClusterURL = strings.TrimSpace(getDefaultString(EnvGrafanaInClusterURL, ""))
 	c.ExternalServices.Grafana.URL = strings.TrimSpace(getDefaultString(EnvGrafanaURL, ""))
-	c.ExternalServices.Grafana.Namespace = strings.TrimSpace(getDefaultString(EnvGrafanaNamespace, c.IstioNamespace))
-	c.ExternalServices.Grafana.Service = strings.TrimSpace(getDefaultString(EnvGrafanaService, "grafana"))
-	c.ExternalServices.Grafana.APIKey = strings.TrimSpace(getDefaultString(EnvGrafanaAPIKey, ""))
-	c.ExternalServices.Grafana.Username = strings.TrimSpace(getDefaultString(EnvGrafanaUsername, ""))
-	c.ExternalServices.Grafana.Password = strings.TrimSpace(getDefaultString(EnvGrafanaPassword, ""))
-	if c.ExternalServices.Grafana.Username != "" && c.ExternalServices.Grafana.Password == "" {
-		log.Error("Grafana username (\"GRAFANA_USERNAME\") requires that Grafana password (\"GRAFANA_PASSWORD\") is set.")
-	}
-	c.ExternalServices.Grafana.InsecureSkipVerify = getDefaultBool(EnvGrafanaInsecureSkipVerify, false)
+	c.ExternalServices.Prometheus.Auth = getAuthFromEnv("GRAFANA")
```

Comment:
Is this intentional? Because it's overwriting prometheus auth config in line 277.
i think it should be 
```
c.ExternalServices.Grafana.Auth = getAuthFromEnv("GRAFANA")
```

Reply:
Excellent question :)

Reply:
oh, good catch!

Reply:
See #1197

Reply:
Thank you for quick reply and PR! :)